src/bindings.cpp

```diff
@@ -0,0 +1,11 @@
+#include <emscripten/bind.h>
+
+#include "ClpIrV1Decoder.hpp"
+
+EMSCRIPTEN_BINDINGS(DecoderModule) {
+    emscripten::class_<ClpIrV1Decoder>("ClpIrV1Decoder")
+            .constructor(&ClpIrV1Decoder::create, emscripten::allow_raw_pointers())
+            .function("estimatedNumEvents", &ClpIrV1Decoder::get_estimated_num_events)
+            .function("buildIdx", &ClpIrV1Decoder::build_idx)
+            .function("decode", &ClpIrV1Decoder::decode);
```

Comment:
I'll more into generating TypeScript definition files with Emscripten (if will we ever find that meaningful after we see the generated files, since our class only implements the abstract class `Decoder` and we have the docs there already. Let's continue the discussion at https://github.com/y-scope/clp-ffi-js/pull/1/files#r1675203013 ).